tests/ui/needless_continue.stderr

```diff
@@ -47,16 +45,15 @@ LL | |             println!("Jabber");
 LL | |         }
    | |_________^
    |
-   = help: Consider dropping the `else` clause, and moving out the code in the `else` block, like so:
-           if (zero!(i % 2) || nonzero!(i % 5)) && i % 3 != 0 {
-               continue;
-           }
-           println!("Blabber");
-           println!("Jabber");
-           ...
-
-error: This `else` block is redundant.
+   = help: consider dropping the `else` clause
+                   if (zero!(i % 2) || nonzero!(i % 5)) && i % 3 != 0 {
+                       continue;
+                   } {
+                       println!("Blabber");
```

Comment:
This seems to be a regression as it now wraps the else part in a block?

Reply:
Yeah, it changed a little bit, but I would keep it that way for two reasons:
1. It's too hard to align this correctly after removing the block
2. Suggesting it like this prevents a bug, that could be produced by the suggestion:

```rust
let y: u32 = 23;
if x {
    continue;
} else {
    let y: u32 = 42;
    // ...
}

println!("{}", y); // outputs 23
```

previous, the suggestion would change the output:

```rust
let y: u32 = 23; // best case: a warning for unused var is produced
if x {
    continue;
}
let y: u32 = 42;
// ...

println!("{}", y); // outputs 42
```

now the output stays the same:

```rust
let y: u32 = 23;
if x {
    continue;
} 
{
    let y: u32 = 42;
    // ...
}

println!("{}", y); //outputs 23
```

Reply:
[Playground](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=557334ca1cc44268d5c4649c36a4b266)